src/protocol/thread/bounds.ts

```diff
@@ -26,7 +26,10 @@ export class NodeBoundsFront {
   }
 }
 
-Object.setPrototypeOf(NodeBoundsFront.prototype, new Proxy({}, DisallowEverythingProxyHandler));
+Object.setPrototypeOf(
+  NodeBoundsFront.prototype,
+  new Proxy(Object.create(null), DisallowEverythingProxyHandler)
```

Comment:
cc: @loganfsmyth - this fixes the issue I mentioned in a call regarding hot reloading checking exports to see if they were React components which caused this to log `Not Alllowed` errors in the console. I think this is a safe fix but I don't have much background on why this Proxy exists to be sure.